tests/AdminPanel/Symfony/AdminBundle/Tests/Functional/Page/UserListPage.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace AdminPanel\Symfony\AdminBundle\Tests\Functional\Page;
+
+class UserListPage extends BasePage
```

Comment:
I think it would be better to create more generic pages, like `ListPage`, `CreateFormPage`